pulp_2to3_migration/app/models/repository.py

```diff
@@ -56,6 +57,11 @@ class Pulp2Repository(BaseModel):
                                                 on_delete=models.SET_NULL,
                                                 null=True)
 
+    # This is needed for migrating Variants of the DistributionTree
+    pulp3_repository = models.ForeignKey(Repository,
```

Comment:
Do we need a custom migration for users to upgrade from older versions of the migration plugin? (e.g. those who migrated Docker and File already, say with Katello 3.15.)
I guess it's not essential if we are not exposing this field in REST API.
It's just somewhat inconsistent that some migrated repos will have this field set and some won't.

Reply:
This field is only used by the RPM plugin. So I am not sure we need to do anything custom here.